alleycats-core/src/test/scala/alleycats/SyntaxSuite.scala

```diff
@@ -54,6 +54,21 @@ object SyntaxSuite {
     val a1: Boolean = x.nonEmpty
   }
 
+  def testOptionEmpty[A: Empty]: Unit = {
```

Comment:
Where is this type parameter used? I don't see it. Can we remove it?

Reply:
Not sure why I did that 🤦🏻‍♂️